hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -537,6 +537,9 @@
       clock_group: "infra",
       reset_connections: {rst_ni: "lc", rst_otp_ni: "lc_io_div4"},
       base_addrs: {core: "0x41000000", prim: "0x41008000", mem: "0x20000000"}
+      param_decl: {
+        ProgFifoDepth: "4",
```

Comment:
@TimEwins i would like to reduce the read fifo size too. But this will have a slight impact on your test.
let me know if it's okay to change.